cel/README.md

```diff
@@ -37,231 +48,357 @@ kubectl apply --filename https://storage.cloud.google.com/tekton-releases-nightl
 
 ## Usage
 
-To evaluate a CEL expressions using `Custom Tasks`, we need to define a [`Run`](https://github.com/tektoncd/pipeline/blob/master/docs/runs.md)
-type with `apiVersion: cel.tekton.dev/v1alpha1` and `kind: CEL`. The `Run` takes the CEL expressions to be evaluated
-as `Parameters`. If executed successfully, the `Run` will produce the evaluation results as `Results` with names corresponding
-with the `Parameters` names. See the [examples](examples) folder for `CEL` `Custom Tasks` to run or use as samples. 
+To evaluate a CEL expressions using `Custom Tasks`, we need to define a [`Run`][run] type with 
+`apiVersion: custom.tekton.dev/v1alpha1` and `kind: CEL`. 
+
+The `Run` takes the CEL expressions to be evaluated through `expressions` field. The `Run` optionally takes CEL 
+environment variables through the `variables` field.
+
+If executed successfully, the `Run` will produce the evaluation results as `Results` with names corresponding
+with the `Expressions`'s names. See the [examples](examples) folder for `CELs` to run or use as samples. 
+
+### Configuring a `CEL`
 
-### Configuring a `CEL` `Custom Task`
+A `CEL` definition supports the following fields:
 
-The `CEL` `Custom Task` is defined in a `Run`, which supports the following fields:
+- Required:
+  - [`apiVersion`][kubernetes-overview] - Specifies the API version, `custom.tekton.dev/v1alpha1`.
+  - [`kind`][kubernetes-overview] - Identifies this resource object as a `CEL` object.
+  - [`metadata`][kubernetes-overview] - Specifies the metadata that uniquely identifies the `CEL`, such as a `name`.
+  - [`spec`][kubernetes-overview] - Specifies the configuration for the `CEL`.
+    - [`expressions`](#specifying-cel-expressions) - Specifies the CEL expressions to be evaluated
+- Optional:
+    - [`variables`](#specifying-cel-environment-variables) - Specifies the CEL environment variables 
```

Comment:
nit pick: i think it would be clearer to list variables directly under spec e.g. something like:

```
    - [`expressions`](#specifying-cel-expressions) - Specifies the CEL expressions to be evaluated
    - [`variables`](#specifying-cel-environment-variables) -  (optional) Specifies the CEL environment variables 
```

with the current version its not obvious that variables is under spec? just my opinion tho XD

Reply:
makes sense, fixed it :)